symfony/plugins/orangehrmPerformancePlugin/entity/Kpi.php

```diff
@@ -2,76 +2,182 @@
      */
-    private $id;
-
-    /**
-     * @var string
-     *
-     * @ORM\Column(name="job_title_code", type="string", length=10)
-     */
-    private $jobTitleCode;
+    private int $id;
 
     /**
      * @var string
      *
      * @ORM\Column(name="kpi_indicators", type="string", length=255)
      */
-    private $kpi_indicators;
+    private string $title;
 
     /**
      * @var int
      *
      * @ORM\Column(name="min_rating", type="integer", length=11)
      */
-    private $min_rating;
+    private int $minRating;
 
     /**
      * @var int
      *
      * @ORM\Column(name="max_rating", type="integer", length=11)
      */
-    private $max_rating;
+    private int $maxRating;
 
     /**
-     * @var int
+     * @var int|null
      *
-     * @ORM\Column(name="default_kpi", type="integer", length=2)
+     * @ORM\Column(name="default_kpi", type="integer", length=2, nullable=true)
      */
-    private $default_kpi;
+    private ?int $defaultKpi = null;
 
     /**
-     * @var \Doctrine\Common\Collections\Collection
+     * @var DateTime|null
      *
-     * @ORM\OneToMany(targetEntity="OrangeHRM\Entity\JobTitle", mappedBy="Kpi")
-     * @ORM\JoinColumns({
-     *   @ORM\JoinColumn(name="jobTitleCode", referencedColumnName="id")
-     * })
+     * @ORM\Column(name="deleted_at", type="datetime")
+     */
+    private ?DateTime $deletedAt = null;
+
+    /**
+     * @var JobTitle|null
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\JobTitle")
+     * @ORM\JoinColumn(name="job_title_code", referencedColumnName="id", nullable=true)
+     */
+    private ?JobTitle $jobTitle = null;
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
      */
-    private $JobTitle;
+    public function setId(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTitle(): string
+    {
+        return $this->title;
+    }
 
     /**
-     * Constructor
+     * @param string $title
      */
-    public function __construct()
+    public function setTitle(string $title): void
     {
-        $this->JobTitle = new \Doctrine\Common\Collections\ArrayCollection();
+        $this->title = $title;
     }
 
+    /**
+     * @return int
+     */
+    public function getMinRating(): int
+    {
+        return $this->minRating;
+    }
+
+    /**
+     * @param int $minRating
+     */
+    public function setMinRating(int $minRating): void
+    {
+        $this->minRating = $minRating;
+    }
+
+    /**
+     * @return int
+     */
+    public function getMaxRating(): int
+    {
+        return $this->maxRating;
+    }
+
+    /**
+     * @param int $maxRating
+     */
+    public function setMaxRating(int $maxRating): void
+    {
+        $this->maxRating = $maxRating;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getDefaultKpi(): ?int
```

Comment:
```suggestion
    public function isDefaultKpi(): bool
```

Reply:
Changed to `public function isDefaultKpi(): ?bool`